atef/config.py

```diff
@@ -52,6 +52,53 @@ class Configuration:
     #: Tags tied to this configuration.
     tags: Optional[List[str]] = None
 
+    def children(self) -> List[Any]:
+        """ Return children of this group, as a tree view might expect """
```

Comment:
nitpick: lingering spacy comments in this file

Reply:
How did this escape the filter!?  (answer: my regex was bad)